articles/hdinsight/hdinsight-hadoop-install-presto.md

```diff
@@ -22,7 +22,7 @@ HDInsight also offers the Starburst Presto application for Apache Hadoop cluster
 > The steps in this article require an HDInsight 3.5 Hadoop cluster that uses Linux. Linux is the only operating system used on HDInsight version 3.4 or later. For more information, see [HDInsight versions](hdinsight-component-versioning.md).
 
 ## What is Presto?
-[Presto](https://prestodb.io/overview.html) is a fast-distributed SQL query engine for big data. Presto is suitable for interactive querying of petabytes of data. For more information on the components of Presto and how they work together, see [Presto concepts](https://github.com/prestodb/presto/blob/master/presto-docs/src/main/sphinx/overview/concepts.rst).
+[Presto](https://prestosql.io/overview.html) is a fast-distributed SQL query engine for big data. Presto is suitable for interactive querying of petabytes of data. For more information on the components of Presto and how they work together, see [Presto concepts](https://prestosql.io/docs/current/overview/concepts.html).
```

Comment:
This link doesn't work:
https://prestosql.io/overview.html

Reply:
Sorry about that. When we redid the website we got rid of some obsolete content. I'll submit another PR to fix the broken ones.